R/bambu.R

```diff
@@ -103,31 +110,47 @@ bambu <- function(reads = NULL, rcFile = NULL, rcOutDir = NULL,
             rm.readClassSe <- TRUE # remove temporary read class files 
         }
         readClassList <- bambu.processReads(reads, annotations, 
-            genomeSequence = genome, 
+            genomeSequence = genomeSequence, 
             readClass.outputDir = rcOutDir, yieldSize, 
-            bpParameters, stranded, verbose,
-            min.readCount = isoreParameters[["min.readCount"]])
+            bpParameters, stranded, verbose)
     } else {
         readClassList <- rcFile
     }
-    if (discovery) {
+    if (!quantOnly) {
+        defaultModels = readRDS(system.file("extdata", "defaultModels.rds",
```

Comment:
you can have this as internal data, by adding the code to data-raw/DATASET.R by adding this to the file before line 99
`defaultModels = readRDS(system.file("extdata", "defaultModels.rds",package = "bambu")`

and then add this to line 99 
`usethis::use_data(..., defaultModels)`